.changes/1771-fix-typing-notifications.md

```diff
@@ -0,0 +1,2 @@
+- [fix] do not send typing notifications if the user disabled that in the settings
+- [fix] only one room could be receiving typing state at a time despite multiple rooms actually being in that state.
```

Comment:
Does that mean we don't get notified about typing states if happening across multiple rooms? Or they would be queued up?

Reply:
> Does that mean we don't get notified about typing states if happening across multiple rooms?

Yes, previously only one room would be shown as typing on the flutter side ever. I noticed in the code ...